src/templates/ObservabilityPackDetails.js

```diff
@@ -226,6 +226,7 @@ const ObservabilityPackDetails = ({ data, location }) => {
             <Tabs.BarItem id="alerts" count={pack.alerts?.length ?? 0}>
               Alerts
             </Tabs.BarItem>
+            <Tabs.BarItem id="data-sources">Data sources</Tabs.BarItem>
```

Comment:
I think we want to include the count of data sources in this tab title, similar to the other tabs. Worth mentioning that this is the combination of the `instrumentation` and `documentation` fields

Reply:
I was wondering about this, it felt kind of weird to count those as individual items but i'm happy to add if that makes sense to you

Reply:
Ye I agree just to keep it consistent with the design of the other tabs (though also think weird)